lib/iris/experimental/ugrid/__init__.py

```diff
@@ -3274,6 +3279,135 @@ def context(self):
 PARSE_UGRID_ON_LOAD = ParseUGridOnLoad()
 
 
+def _meshes_from_cf(cf_reader):
+    """
+    Common behaviour for extracting meshes from a CFReader.
+
+    Simple now, but expected to increase in complexity as Mesh sharing develops.
+
+    """
+    # Mesh instances are shared between file phenomena.
+    # TODO: more sophisticated Mesh sharing between files.
+    # TODO: access external Mesh cache?
+    mesh_vars = cf_reader.cf_group.meshes
+    meshes = {
+        name: _build_mesh(cf_reader, var, cf_reader.filename)
+        for name, var in mesh_vars.items()
+    }
+    return meshes
+
+
+def load_mesh(uris, var_name=None):
+    """
+    Create a single :class:`Mesh` object from one or more NetCDF files.
```

Comment:
Small point, but I think 
```suggestion
    Load a single :class:`Mesh` object from one or more NetCDF files.
```
is a more suggestive of what is does with the input source(s)

Reply:
ec55e07404debf2623283b28b6591c5bea4ae39c